documentation/book/assembly-using-the-kafka-bridge.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+[id='assembly-using-the-kafka-bridge-{context}']
+= Using the {ProductName} Kafka Bridge
+
+This chapter provides an overview of the {ProductName} Kafka Bridge and helps you get started with using the API. To view the API documentation including example requests and responses, see LINK **{ProductName} Kafka Bridge API resources**. For information on how to configure the Kafka Bridge after deployment, see LINK **{ProductName} Kafka Bridge Configuration**.
+
+ifdef::TechPreview[]
+IMPORTANT: The {ProductName} Kafka Bridge is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service-level agreements (SLAs) and might not be functionally complete; therefore, Red Hat does not recommend implementing any Technology Preview features in production environments. This Technology Preview feature provides early access to upcoming product innovations, enabling you to test functionality and provide feedback during the development process. For more information about support scope, see {TechPreviewFeatures} on the Red Hat Customer Portal.
+endif::TechPreview[]
```

Comment:
I would mention that with TP the API could change in the future and even breaking changes could be possible.

Reply:
OK, but this is standard boilerplate text. I would need to find out if we are allowed to change it.

Reply:
I guess the note about breaking changes also applies to the upstream docs?
I've removed the TP notice from the upstream.